codex_of_the_damned/templates/archetypes/bibliodeque/index.html

```diff
@@ -0,0 +1,334 @@
+{% extends "layout.html" %}
+{% import "_layout_sponsor.html" as sponsor %}
```

Comment:
I would consider adding this to `archetypes/index.html` directly, what do you think ?

Reply:
Are you talking about the index file or something else?

Reply:
resolved